arangod/VocBase/LogicalDataSource.h

```diff
@@ -24,18 +24,20 @@
 #pragma once
 
 #include <atomic>
-#include <string_view>
+#include <cstdint>
+#include <string>
 
 #include "Basics/Result.h"
+#include "Basics/debugging.h"
```

Comment:
```suggestion
#include "Assertions/Assert.h"
```

Btw for TRI_ASSERT now can be included only it.
Thanks to @markuspf